minetest/minetest
# Support generation of working Xcode project for signature purposes on MacOS

This allows `cmake -G Xcode` to generate useful Xcode projects for build and also archive, which have everything needed to go through the Apple notarization process.

Related to #14939.

## To do

This PR is a Ready for Review

## How to test

Generate Xcode project and try to build and archive. Check the results.

To prevent problems with signing dylib, I am using a build with static libraries, so I do not care about dylibs.

## Comment

Irrlichtmt CMake has to be changed due to go around cmake Xcode generator bug.
